tsconfig.json

```diff
@@ -13,16 +13,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "baseUrl": "./",
-    "paths": {
-      "@components/*": ["src/components/*"],
-      "@utils/*": ["src/utils/*"],
-      "@hooks/*": ["src/hooks/*"],
-      "@context/*": ["src/context/*"],
-      "@lib/*": ["src/lib/*"],
-      "@content/*": ["content/*"],
-      "@types": ["src/types.ts"]
-    }
+    "baseUrl": "./src/"
```

Comment:
Pointing `baseUrl` to `src` allows us to delete all of the `paths` config.

All of the other file changes are removing the `@` from the beginning of the module import.